mods/d2k/rules/arrakis.yaml

```diff
@@ -9,8 +9,7 @@ spicebloom.spawnpoint:
 		GrowthTerrainTypes: SpiceSand
 		SpawnActor: spicebloom
 		GrowthSequences: grow0
-		GrowthDelay: 250, 750
-		RespawnDelay: 1, 2
+		GrowthDelay: 1750, 3250
```

Comment:
Oh, I see.  I was under the impression that `GrowthDelay` was the delay between growth steps, but it's actually the total growth time.  Would `Lifetime` be a better name here?